door_ivr/door_ivr.test.conf

```diff
@@ -1,5 +1,6 @@
 [backend]
-api_url=http://127.0.0.1:3000/api
+auth_api_url=http://127.0.0.1:3000/api
+door_api_url=http://127.0.0.1:3001/api
```

Comment:
Currently the fauna mock is on 3001 - maybe move the mock to 3002 and leave 3002 in the tests?

Reply:
Sounds like a plan